PyInstaller/hooks/hook-flexx.py

```diff
@@ -0,0 +1,13 @@
+import os
+from PyInstaller.utils.hooks import collect_data_files, get_module_file_attribute
+
+datas = collect_data_files('flexx', include_py_files=True)
```

Comment:
Please move this down below the `flexx_py_path`. I was confused when reading `datas = datas + ...` in line 10.

Reply:
Done!